### PR TITLE
Fetch attestations from storage in the API.

### PR DESCRIPTION
**Pull request by dlorenc**
Signed-off-by: Dan Lorenc <lorenc.d@gmail.com>

### CODE REVIEW

**Comment by dlorenc** [comment]
TODO:

- [x] Tests
- [x] CLI support
- [x] Media Types?

File: pkg/storage/storage.go [hunk 1]
```diff
@@ -65,3 +66,24 @@ func (b *Blob) StoreAttestation(ctx context.Context, key, attestationType string
 	}
 	return w.Close()
 }
+
+func (b *Blob) FetchAttestation(ctx context.Context, key string) ([]byte, string, error) {
+	log.Logger.Infof("fetching attesattion %s", key)
```
Comment (bobcallaway) [inline pkg/storage/storage.go]: s/attesattion/attestation

**Review by bobcallaway — APPROVED** [review]
LGTM, other than one spelling issue in log message

File: pkg/storage/storage.go (hunk 1 above)
Comment (dlorenc) [reply pkg/storage/storage.go]: Thanks!

**Pull request merged**